requestor/README.md

```diff
@@ -51,8 +51,99 @@ Default name of this subnet is `erigon`.
 
 ## Use requestor server
 
+### Endpoints
+
 All requests must contain a header `{'Authorization': 'Bearer [ETHEREUM_ADDRESS]'}`
 
     GET  /getInstances      - list of all instances created by the user (includes stopped instances)
     POST /createInstance    - create new instance, this requires "params" in body and also accepts optional "name"
     POST /stopInstance/<id> - stop instance with id <id>
+
+### Command-line interface
+
+You can use CLI to interact with the requestor server. First, generate a `keystore.json` file. You can export from yagna or use [vanity-eth](https://vanity-eth.tk/). Then you can list, create and stop erigon instances:
+```shell
+$ python3 -m venv .venv
+$ . .venv/bin/activate
+$ pip install -r requirements.txt
+$ ./erigolem_cli.py --keystore ./keystore.json --password <PASSWORD> list 
+[]
+$ ./erigolem_cli.py --keystore ./keystore.json --password <PASSWORD>  create '{"name": "My Node", "params": {"network": "goerli"}}'
+{'created_at': '2022-03-07T15:35:59.355692', 'id': 'a2303ae2c2de4a83a86a8fa69cb9b303', 'init_params': {'network': 'goerli'}, 'name': 'My Node', 'status': 'pending'}
```

Comment:
Is it just raw python output? It doesn't wark nicely with `jq` because of `'` instead of `"` 😿 
It'd be great if one just could pipe through `jq` for readability. WDYT ❓

Reply:
Cool, I'll make it proper JSON.

Reply:
I've added a separate task for this: [APPS-559](https://golemproject.atlassian.net/browse/APPS-559)